packages/react-devtools-extensions/src/__tests__/updateMockSourceMaps.js

```diff
@@ -139,12 +199,38 @@ function compile(fileName) {
     // the second item.
     x_facebook_sources: [[null, [encodedHookMap]]],
   };
+  const fbSourcesExtendedIndexMap = {
+    version: fbSourcesExtendedSourceMap.version,
+    file: fbSourcesExtendedSourceMap.file,
+    sections: [
```

Comment:
Ideally I would add a leading section to this fixture, and add the corresponding amount of whitespace (spaces / newlines) to the code, to exercise the logic for multiple sections.